mautic/mautic
# Updated icons for Segment Filter List

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This pull request addresses the user interface of the segment filter list by removing irrelevant icons. 




| Before                                 | After
| -------------------------------------- | ---
|                <img width="354" alt="1 - Before Filter" src="https://github.com/user-attachments/assets/73f64a07-dadf-4c74-87ca-bf8e79ae8674">                        |  <img width="349" alt="3 - After filters - Contact" src="https://github.com/user-attachments/assets/a813442c-3b92-47ea-bf59-d931d6d0f495"> <img width="356" alt="3 - After filters - Company" src="https://github.com/user-attachments/assets/5148765a-a69d-4681-8c7b-29fa827e3ae8"> <img width="353" alt="3 - After filters - Behavior" src="https://github.com/user-attachments/assets/4434491e-1e69-48d3-a5f6-3c5cd97cefd6">
|<img width="555" alt="5 - Before" src="https://github.com/user-attachments/assets/855f02a2-6a92-459b-b319-6f6044e83339">|<img width="471" alt="5 - After Filter widget" src="https://github.com/user-attachments/assets/5cbbc3ac-6faa-4219-9bd7-9eab50dabae6">



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to segment, and click on New
3. Go to the filters tab
4. Search for Contact Behavior and Actions. The section is not have the icon
5. Select one from the list, e.g., Asset Download; the added filter field should be without an icon.
